spec/fixtures/lua_shared_dict

```diff
@@ -0,0 +1,25 @@
+kong 5m
```

Comment:
An idea:
Could we change this file into a lua module which will return a string (like we did in busted code),
then we can just write in busted:
```
resty_flags = resty_flags .. require("spec.fixtures.lua_shared_dict")
```

Reply:
good idea, I'll modify it later.

Reply:
done